src/cheri/xycheriot.adoc

```diff
@@ -0,0 +1,678 @@
+a constant {cheriot_ycfg_value} in `mycfg`.
+(See <<uycfg>> for the details of this CSR.)
+
+[NOTE]
+=====
+TODO: That value is, of course, not finalized.
+=====
+
+==== Bounds Encoding
+
+CHERIoT's bounds encodings use 9-bit top and bottom mantissa fields
+and a compressed 4-bit exponent value.
+The resulting encoding significantly differs from the bounds
+of the <<app_cap_description>> encoding; CHERIoT's encoding...
+
+- has byte alignment only up to length 511.
+
+- does not guarantee a representable region beyond the authorized base and top
+  (except for "one past the end", where the address is equal to the limit).
++
+This requires using a non-default
+<<section_xycheriot_auipc_shift,AUIPC shift>> value.
+
+[NOTE]
+=====
+TODO: As in §7.13.3 / "Capability bounds" in {cheriot_v1_isa}
+=====
+
+==== Permissions
+
+{cheriot_ext_name} defines _one_ <<sec_cap_sdp,software defined permission>>,
+which we call `U0`.
+That is, its `SDPLEN` value is `1`.
+
+There are 11 other architectural permission bits in {cheriot_ext_name},
+all of which are from its constituent components:
+
+* <<r_perm>>, <<w_perm>>, <<c_perm>>, <<lm_perm>>, <<x_perm>>, and <<asr_perm>>
+  from {cheri_base_ext_name};
+
+* <<zylevels1_lg_perm>>, <<zylevels1_sl_perm>>, and <<zylevels1_gl_perm>>
+  from <<section_zylevels1>>; and
+
+* <<zyseal_se_perm>> and <<zyseal_us_perm>> from <<section_zyseal>>.
+
+The aggregate permissions bit field (<<acperm_bit_field>>) used by <<GCPERM>>
+and <<CLRPERM>> for {cheriot_ext_name} is thus:
+include::./img/xycheriot-acperm_bit_field.edn[]
+
+[#section_xycheriot_roots]
+==== Roots
+
+{cheriot_ext_name} defines _three_ <<root-cap>> capability values:
+
+* Its <<root-rx-cap>> capability value is defined as
+  having <<zylevels1_gl_perm>> set and granting exactly all of
+  <<x_perm>>, <<r_perm>>, <<c_perm>>, <<lm_perm>>, <<asr_perm>>,
+  and <<zylevels1_lg_perm>>.
+
+* Its <<root-rw-cap>> capability value is defined as
+  having <<zylevels1_gl_perm>> set and granting exactly all of
+  <<r_perm>>, <<w_perm>>, <<c_perm>>, <<lm_perm>>,
+  <<zylevels1_lg_perm>>, and <<zylevels1_sl_perm>>.
+
+* It defines a _root sealing_ capability value that is unsealed,
+  has full address space bounds,
+  and grants exactly all of <<zyseal_se_perm>>, <<zyseal_us_perm>>, and `U0`.
+
+These root capabilities satisfy the following properties by construction.
+They will necessarily remain true through any series of <<CLRPERM>> transitions:
+
+* At most one of <<x_perm>> or <<w_perm>> may be set.
+
+* The <<zyseal_se_perm>>, <<zyseal_us_perm>>, and `U0`
+  collectively conflict with
+  either of the <<r_perm>> and <<w_perm>> permissions.
++
+That is, a capability may grant permissions
+from at most one of these two sets;
+this serves to partition capabilities that refer to memory addresses
+from those that refer to <<sec_cap_type>> values
+(those granting <<zyseal_se_perm>> and/or <<zyseal_us_perm>>)
+or uninterpreted integers (those granting `U0`).
```

Comment:
I think all this is probably unnecessary for this draft; writing this down ties us down to this encoding right away.